### ISSUE TITLE
Unable to Sign In with LinkedIn

### ISSUE THREAD

**Issue by Joey-Davis** [FreeCodeCamp/FreeCodeCamp#4291]
Yesterday and today I have been given an error message when attempting to sign in using my LinkedIn account (as I usually do). I have not made any modifications to my LinkedIn account/settings nor to any settings in FreeCodeCamp.

If it helps narrow down where the issue might be -- my last successful login was on November 2nd.

Screenshot of error message (after LinkedIn redirects to FreeCodeCamp):
![screenshot](https://cloud.githubusercontent.com/assets/11513288/11078648/447a41a2-87d5-11e5-988d-61ee332c2eb8.png)

**Comment by faizaanceg** [comment]
Can you try this with other browsers? Maybe refresh cache? I just tried with my LinkedIn and it worked fine.

**Comment by Joey-Davis** [comment]
I've cleared the cache, tried a different browser, and attempted from 2 other computers -- all rendering the same result:

![screenshot](https://cloud.githubusercontent.com/assets/11513288/11079185/8a785ff6-87d8-11e5-9a10-17e321620bc8.png)

**Comment by BerkeleyTrue** [comment]
@Joey-Davis Are you able to signout and signin on linkedin?

**Comment by Joey-Davis** [comment]
@BerkeleyTrue Yes, I am able to sign in and out of LinkedIn without a problem. Before trying to log into FreeCodeCamp on Monday, I had not signed in or used LinkedIn since Nov 2 either. FreeCodeCamp is still listed as an "Authorized External Application" on my LinkedIn settings as it was before.

**Comment by BerkeleyTrue** [comment]
@Joey-Davis What is your FCC username?

**Comment by Joey-Davis** [comment]
@BerkeleyTrue My FCC username is joey_davis.

I took a look through FCC's source code this evening and noticed that the "forgot password" option, only available from the email and password sign-in page, checks if the email provided exists in the database but not which provider is used for authentication. Using this method, I was able to set a password and regain access to my account, given that FCC had my email from the LinkedIn credentials. 

I am still unable to log in using my LinkedIn account but this is no longer an immediate issue for me (since I can log in with an alternate method now).

Thanks for your help!
